crates/ra_parser/src/grammar.rs

```diff
@@ -119,7 +119,22 @@ pub(crate) fn meta_item(p: &mut Parser) {
             items::token_tree(p);
             break;
         } else {
-            p.bump();
+            // https://doc.rust-lang.org/reference/attributes.html
+            // https://doc.rust-lang.org/reference/paths.html#simple-paths
+            // The start of an meta must be a simple path
+            match p.current() {
+                IDENT | COLONCOLON | SUPER_KW | SELF_KW | CRATE_KW => p.bump(),
+                EQ => {
+                    p.bump();
+                    match p.current() {
+                        c if c.is_literal() => p.bump(),
+                        TRUE_KW | FALSE_KW => p.bump(),
+                        _ => {}
+                    }
+                    break;
+                }
+                _ => break,
+            }
```

Comment:
Let's add a test for it?

Also, this was recently changed in https://github.com/rust-lang/rust/pull/57367/, I am not sure if the refercne is up-to-date

Reply:
I don't know where to add the test, as this function only called from mbe right now.
I can add that test in mbe if you want.

Reply:
Ah, right, the parser doesn't need this itself. I guess I am fine with existing tests in mbe then!